industrial_extrinsic_cal/yaml/ical_srv_target.yaml

```diff
@@ -0,0 +1,12 @@
+---
+moving_targets:
+-
+     target_name: target2
+     target_type: 2
+     circle_dia:  0.5
+     spacing:  0.5
+     target_frame: target_frame
+     transform_interface: ros_lti
+     parent_frame: camera_frame
+     target_rows: 7
+     target_cols: 7
```

Comment:
@lawrenceglewis

Reply:
What is the problem?

Reply:
@drchrislewis i've made my changes please review again.